app/views/settings/change_password.html.erb

```diff
@@ -0,0 +1,45 @@
+<!-- Tab navbar -->
+<div class="sticky-top">
+  <!-- General Nav bar -->
+  <nav class="navbar navbar-expand-lg navbar-light bg-dark p-0">
+    <div class = "container-fluid">
```

Comment:
please search-and-replace ALL instances of ' = ' with '=' in HTML markup.  this should be caught by a lint pass